examples/client_data.c

```diff
@@ -8,6 +8,8 @@
 #include <unistd.h>
 #include <uv.h>
 
+#define QUOTE(...) #__VA_ARGS__
```

Comment:
@bozakov: the use of macro for passing on the `config_property` is inconsistent across this commit. Since it adds to the readability, better to use it everywhere IMO.

Reply:
fixed